ex_functions/ex_csv_parser.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+DEFAULT_INPUT_SEPARATOR = ";"
+DEFAULT_OUTPUT_SEPARATORS = { "between_columns": "|", "between_rows": "-", "between_columns_and_rows": "+" }
+
+
+# Read the input file and transform it into a list of lists
+# The outer list contains rows, and the inner lists contain values
+def load_input_file(input_file_path: str, input_separator=";") -> list:
+    return [ line.rstrip("\n").split(input_separator) for line in open(input_file_path, "r") ]
+
+
+# Pretty-print a list of lists
+def print_output(rows: list, output_separators: list) -> None:
+    max_column_lengths = find_max_column_lengths(rows)
```

Comment:
É unha sola función, pero quizá estaría ben subir esta función un nivel de abstracción e retirala de print_output, así quedaría 1: leer 2: procesor 3: exportar e nos quedan os 3 niveis  típicos(ETL, MVC. interfaz, lóxica e persistencia). 
Isto permitiría medrar o procesado e a lóxica de saída independentemente, sen tocar nada máis.